src/rpc/blockchain.cpp

```diff
@@ -1508,6 +1509,7 @@ static UniValue getmempoolinfo(const JSONRPCRequest& request)
                 {},
                 RPCResult{
             "{\n"
+            "  \"loaded\": true|false         (boolean) True if the mempool has been loaded/initialized.\n"
```

Comment:
Should clarify that this means loaded *from disk* and if -persistmempool is on?

Reply:
If `-persistmempool` is off, `g_is_mempool_loaded` is set at the end of `ThreadImport`, after block import. Ideas on how to articulate that?

Reply:
How about this: https://github.com/bitcoin/bitcoin/pull/15323/commits/fce31d4d96704dde653e2c19091ff2e703f28323

Reply:
nit, why `/initialized`?

Reply:
Because it will also be true in cases where it has not been loaded from disk, but only "initialized". Maybe not worth mentioning.

Reply:
Would prefer something like
`True if the mempool has been loaded from disk (if persistent)`

Reply:
I think `True if the mempool has been loaded from disk (if persistent), or need not be` is much less clear than the more simple `True if the mempool is fully loaded.` The first raises more questions than it answers ("What is persistent? When need it not be?")

Reply:
Maybe `s/persist/-persistmempool/g`?